gprofiler/profilers/java.py

```diff
@@ -277,12 +272,46 @@ def get_java_version(process: Process, stop_event: Event) -> str:
     # 2. assume JDK type by the path, e.g the "java" Docker image has
     #    "/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java" which means "OpenJDK". needs to be checked for
     #    other JDK types.
-    java_path = f"/proc/{nspid}/exe"
+    java_path: Optional[str] = None
```

Comment:
I think we can ditch the attempt to have a single return-point in this function - remove this var, use `return` wherever assigning to it, and add `return None` last

Reply:
👍